pdc_client/tests/component/data/global_component/detail.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "1",
-    "name": "Test Global Component",
+    "name": "Test_Global_Component",
```

Comment:
It's not `pdc` splitting it into 3 parts, that is done by Bash. If you want a single strings with spaces, quoting is required: `pdc global-component update --name "new test name" "Test Global Component"`.